extensions/wikia/TemplateDraft/controllers/TemplateDraftController.class.php

```diff
@@ -59,4 +59,43 @@ public function approveDraft( Title $title ) {
 		// Save to parent page
 		$page->doEdit( $draftContent, wfMessage( 'templatedraft-approval-summary' )->inContentLanguage()->plain() );
 	}
+
+	public function markTemplateAsNotInfobox() {
+		/**
+		 * First, validate a request
+		 */
+		if ( !$this->wg->User->isPowerUser()
+			|| !$this->isValidPostRequest()
+		) {
+			$this->response->setVal( 'status', false );
+			return false;
+		}
+
+		/**
+		 * Then check the pageId param
+		 */
+		$pageId = $this->getRequest()->getInt( 'pageId' );
+		if ( $pageId === 0 ) {
+			$this->response->setVal( 'status', false );
+			return false;
+		}
+
+		/**
+		 * Wikia::setProps unfortunately fails silently so if we get to this point
+		 * we can set the response's status to true anyway...
+		 */
+		$this->response->setVal( 'status', true );
+
+		Wikia::setProps( $pageId, [
+			self::TEMPLATE_INFOBOX_PROP => 0,
```

Comment:
Since #7660 is not ready to be merged yet I'd suggest leaving it as is and ensure consistency when working on #7660 .